R/dist_spec.R

```diff
@@ -698,11 +699,12 @@ plot.dist_spec <- function(x, samples = 50L, res = 1, cumulative = TRUE, ...) {
 
   # Plot PMF and CDF as facets in the same plot
   plot <- ggplot(
-    pmf_data, mapping = aes(x = x, y = p, group = sample, linetype = type)
+    pmf_data, mapping = aes(x = x, y = p, group = sample, color = type)
```

Comment:
This will always print colour, even if just one type is needed - just double checking that's what you want (it's fine by me).

Reply:
Yes, this was intentional.